_pages/play.html

```diff
@@ -6,9 +6,6 @@
 ---
 
 <div class="container my-4">
-  <div class="alert alert-danger" role="alert">
-    <strong>Note</strong> Server status is temporarily disabled! You can still play using version 1.8-1.15.2.
-  </div>
```

Comment:
Instead of removing this, can we change it to a `alert-warning` and change the text to something like `Some features are temporarily disabled!`? Or if it's *only* map tags, we can say `Map tags are temporarily disabled!`.